src/components/TeamSidebar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { ListGroup } from 'react-bootstrap';
+
+class TeamSidebar extends React.Component {
+
+    render() {
+        const { teamData, selectedTm, setSelectedTm } = this.props;
+
+        return (
+            <ListGroup>{ Object.keys(teamData).map((tmKey, idx) => {
+                    const { name } = teamData[tmKey];
+
+                    return <ListGroup.Item key={ idx } 
+                                           onClick={ () => { setSelectedTm(tmKey) } }
+                                           active={ tmKey == selectedTm ?? 'true' }>
```

Comment:
Adding a `style={{cursor: 'pointer'}}` prop here would make it more apparent to users that it's a clickable tab.